### PR TITLE
Typing ~ should produce \sim

### PR DESCRIPTION
**Pull request by laughinghan**
Note that `~` is a special character in LaTeX, it's a non-breaking space
kinda like `&nbsp;`
http://tex.stackexchange.com/a/41268

### CODE REVIEW

File: src/commands/math/basicSymbols.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ LatexCmds.operatorname = P(MathCommand, function(_) {
 
 LatexCmds.f = P(Letter, function(_, super_) {
   _.init = function() {
-    Symbol.p.init.call(this, this.letter = 'f', '<var class="mq-florin">&fnof;</var>');
+    Symbol.p.init.call(this, this.letter = 'f', '<var class="mq-florin">f</var>');
```
Comment (jwmerrill) [inline src/commands/math/basicSymbols.js]: Did you mean to include this change to f/florin?

Comment (laughinghan) [inline src/commands/math/basicSymbols.js]: Fuck. Nope, great catch

File: src/commands/math/advancedSymbols.js [hunk 2]
```diff
@@ -3,7 +3,6 @@
  ***********************************/
 
 LatexCmds.notin =
-LatexCmds.sim =
```
Comment (jwmerrill) [inline src/commands/math/advancedSymbols.js]: Does this mean that users of stock mathquill will no longer be able to enter this is \sim?

Comment (laughinghan) [inline src/commands/math/advancedSymbols.js]: No, stock MathQuill includes both basic and advanced symbols, `LatexCmds.sim` just got moved to the basic symbols list

**Pull request merged**